### ISSUE TITLE
Fix duplicate `prefer-spread` rule

### ISSUE THREAD

**Issue by nunofgs** [seegno/eslint-config-seegno#21]
```
YAMLException: Cannot read config file: …/node_modules/eslint-config-seegno/.eslintrc
Error: duplicated mapping key at line 163, column 19:
      prefer-spread: 2
```

**Comment by joaogranado** [comment]
This will be fixed with #19, so I'll close this PR.

**Comment by joaogranado** [comment]
Reopening this issue since it's still active and should be closed by a different PR.
